vnext/Microsoft.ReactNative.Cxx/JSI/NodeApiJsiRuntime.h

```diff
@@ -5,13 +5,14 @@
 #ifndef MICROSOFT_REACTNATIVE_JSI_NODEAPIJSIRUNTIME
 #define MICROSOFT_REACTNATIVE_JSI_NODEAPIJSIRUNTIME
 
+// Standard Library
+#include <cstdint>
```

Comment:
I wonder why this change is required. The <cstdint> must be included through the js_native_api.h included by the js_native_ext_api.h

Reply:
This was needed because the ReactNativePicker project (an external npm package thing used in Playground) was building NodeApiJsiRuntime.cpp through a vcxitems import. I'll try to find a cleaner alternative there.